services/workflows-service/src/workflow/workflow.controller.internal.unit.test.ts

```diff
@@ -121,6 +122,7 @@ describe('WorkflowControllerInternal', () => {
             numb: 'context',
           },
         };
+        axios.post = jest.fn().mockResolvedValue({});
```

Comment:
Check out `nock`, `,msw`, and [the following -](https://kentcdodds.com/blog/stop-mocking-fetch) (no need for changes)